client/components/App.js

```diff
@@ -1,6 +1,8 @@
 import React, { PropTypes, Component } from 'react';
 import TextArea from './TextArea.js';
 import Button from './button.js';
```

Comment:
We can delete this line as well as the button.js file since we split it into the two NextButton and PrevButton file.